srcset support #272 (#281)
enonic/doc-xp: docs/api/lib-portal.adoc

! Name  ! Type   ! Attributes ! Default ! Description
! value ! string !            !         ! Html value string to process.
! type  ! string ! <optional> ! server  ! URL type. Either server (server-relative URL) or absolute.

> ! image:xp-770.svg[XP 7.7.0,opts=inline] imageWidths  ! Array.<number> ! <optional> ! server  ! List of image width. Allows to generate image URLs for given image widths and use them in the `srcset` attribute of a `img` tag.

Reviewer: ```suggestion
! image:xp-770.svg[XP 7.7.0,opts=inline] imageWidths  ! Array.<number> ! <optional> ! server  ! List of image widths. Allows to generate image URLs for given image widths and use them in the `srcset` attribute of an `img` tag.
```

Author: done
